addons/dispatch/0.4.x/dispatch-1.yaml

```diff
@@ -0,0 +1,46 @@
+---
+apiVersion: kubeaddons.mesosphere.io/v1beta1
+kind: ClusterAddon
+metadata:
+  name: dispatch
+  labels:
+    kubeaddons.mesosphere.io/name: dispatch
+  annotations:
+    catalog.kubeaddons.mesosphere.io/addon-revision: "0.4.1-1"
+    appversion.kubeaddons.mesosphere.io/dispatch: "0.4.1"
+    endpoint.kubeaddons.mesosphere.io/dispatch: /dispatch/tekton/
+    docs.kubeaddons.mesosphere.io/dispatch: "https://beta-dispatch.d2iq.com/"
+    appversion.kubeaddons.mesosphere.io/argo-cd: "1.2.0"
+    endpoint.kubeaddons.mesosphere.io/argo-cd: /dispatch/argo-cd
+    docs.kubeaddons.mesosphere.io/argo-cd: "https://argoproj.github.io/argo-cd/user-guide/"
+    values.chart.helm.kubeaddons.mesosphere.io/dispatch: "https://raw.githubusercontent.com/mesosphere/charts/master/stable/dispatch/values.yaml"
+spec:
+  namespace: dispatch
+  kubernetes:
+    minSupportedVersion: v1.15.6
+  requires:
+    - matchLabels:
+        kubeaddons.mesosphere.io/provides: storageclass
+  cloudProvider:
+    - name: aws
+      enabled: false
+    - name: azure
+      enabled: false
+    - name: docker
+      enabled: false
+    - name: none
+      enabled: false
+  chartReference:
+    chart: dispatch
+    repo: https://mesosphere.github.io/dispatch
+    version: v0-4-1
```

Comment:
is this version format correct?

Reply:
For now, yes.

Reply:
This will need to be changed as it breaks things elsewhere, https://jira.mesosphere.com/browse/DCOS-62865 has been created as a follow up so that this will be fixed in CI later.